src/Features/Core/Portable/Shared/Extensions/ISymbolExtensions_2.cs

```diff
@@ -246,7 +246,7 @@ private static string GetMethodDocumentation(IMethodSymbol method)
                 case MethodKind.EventRemove:
                 case MethodKind.PropertyGet:
                 case MethodKind.PropertySet:
-                    return method.ContainingSymbol.GetDocumentationComment().SummaryText;
+                    return method.AssociatedSymbol.GetDocumentationComment().SummaryText;
```

Comment:
i'm a little nervous about this somehow being null in some error case... @gafter Is it an invariant that if you're a .PropertyGet/Set that you always have a non-null AssociatedSymbol?  Thanks!

Reply:
(I've forwarded this question to the team internally.)

Reply:
I believe so.  Having said that, extra null checks couldn't hurt.